components/feature/awesomebar/src/main/java/mozilla/components/feature/awesomebar/provider/SearchSuggestionProvider.kt

```diff
@@ -37,7 +37,7 @@ class SearchSuggestionProvider private constructor(
     @VisibleForTesting internal val engine: Engine? = null,
     private val icon: Bitmap? = null,
     private val showDescription: Boolean = true,
-    private val filterExactMatch: Boolean = false
+    private val filterExactMatch: Boolean = false,
```

Comment:
I would avoid all the formatting changes here since they aren't needed for your issue. It's better to preserve the existing git history.